packages/form-system/stories/02-modelvalue.stories.mdx

```diff
@@ -0,0 +1,32 @@
+import { Story, Meta, html } from '@open-wc/demoing-storybook';
+import { render } from '@lion/core';
+import '@lion/input/lion-input.js';
+import { Validator } from '@lion/validate';
+import './helper-wc/h-output.js';
+
+<Meta title="Form Fundaments/Modelvalue" />
+
+# ModelValue
+
+The modelValue or model can be considered as the ‘aorta’ of our form system.
+It is the single source of truth; not only for the current state
+of the form, also for all derived states: interaction, validation, visibility and other states are
+computed from a modelValue change.
+
+## Single source of truth
+
+ModelValues are designed to provide the Application Developer a single way of programmatical
+interaction with the form for an Application Developer.
```

Comment:
```suggestion
ModelValues provide a single way of interacting with a form or its children.
```

Reply:
We mention App Developer here, because Subclassers could have knowledge about/access to ocncepts as formattedValue etc.